packages/msw-addon/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "msw-storybook-addon",
   "description": "Mock API requests in Storybook with Mock Service Worker.",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Please don't bump the package's version, we'll do it for you.

Reply:
cool sorry bout that have removed 👍